examples/nucleo_f072rb/independend_watchdog/main.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023, Zühlke Engineering (Austria) GmbH
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#include <modm/board.hpp>
+#include <modm/platform.hpp>
+
+using namespace Board;
+
+/**
+ * If the button is pressed for more than 4 seconds, the MCU will be reset by the Watchdog.
+ * This can be observed by the faster blinking LED after startup.
```

Comment:
maybe better?: `This can be observed by the LED blinking faster immediately after startup.`